sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_directory_client.py

```diff
@@ -256,8 +257,8 @@ def delete_directory(self, **kwargs):
             This value is not tracked or validated on the client. To configure client-side network timesouts
             see `here <https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/storage/azure-storage-file-datalake
             #other-client--per-operation-configuration>`_.
-        :returns: None.
-        :rtype: None
+        :returns: A response dict.
```

Comment:
Do we have this pattern anywhere else? If this is just placeholder, I would probably just make it match other placeholders i.e. in [_data_lake_file_client.py](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_file_client.py#L501)
```python
:returns: response dict (Etag and last modified).
```

Only reason being, I am not exactly in _love_ with these docstrings, but at least if we make them consistent, we could do a sweeping F+R on the common doc string

Reply:
Sort of an established pattern so I'm trying to establish it :) But this is the most accurate we can get. These operations are returning a `Dict[str, Any]`, and so all of them have `Etag` and `last_modified` in the response dict. Feel free to correct me here if I'm misunderstanding anything, but that's at least what the debugger said.

Reply:
I think what I meant was that the string you are introducing is completely unique i.e. no hits in `main`:
![image](https://github.com/user-attachments/assets/d2a9aeca-9251-4e67-96c0-4aea0878c39e)

And that you may consider changing it to a more common return for these response dicts such as one I found in `_data_lake_file_client.py`, since if you ever wanted to a find-and-replace refactor, it wouldn't find your above string as it would with the suggested change:
![image](https://github.com/user-attachments/assets/acc2b89b-2398-4aa1-a828-38c6f9746de1)

But after doing the actual search query above ^, it doesn't seem to be very common or consistent regardless (only 3 hits) so this is just a super-nit and you can choose to leave as is or take the feedback. But just something to keep in mind in terms of future maintainability!

Reply:
Ah, gotcha. I'll take the feedback and modify the return statement to say "response dict (ETag and last modified)". Some existing docs say "... (Etag and last modified)" but after doing some digging I think it's properly spelled as "ETag".